gameofgame.py

```diff
@@ -86,15 +90,20 @@ def check_guess(guess, country):
         print("You both should learn about capital cities in the world!!!")
         
     # Print the current scores
-    print(f"Current scores: Player 1: {player1_score}, Player 2: {player2_score}")
+    print(
+        f"Current scores: Player 1: {player1_score}, Player 2: {player2_score}"
+        )
     
     # Ask if the players want to continue playing
     play_again = input("Do you want to play again? (y/n) ")
-    if play_again.lower() != "y":
+    if play_again.lower() != "n":
         break
-    
-else:
-    print("The game ended in a tie. Well played both players!")
-    
-print("Thanks for playing!")
-   
\ No newline at end of file
+    # Determine the winner and print the congratulatory message
+    if player1_score > player2_score:
+        winner_message = congrats_message.format(winner=1)
+    elif player2_score > player1_score:
+        winner_message = congrats_message.format(winner=2) 
```

Comment:
The `winner_message`'s here never get printed. So, the players won't see an output message if someone is the winner.